build/messages/src/Messages.json

```diff
@@ -0,0 +1,18 @@
+{
+  "integrations": {
+    "defaultMessage": "Integrations",
+    "description": "Application name"
+  },
+  "integrationsEmptyStateTitle": {
+    "defaultMessage": "No integrations",
+    "description": "Integrations Empty State title"
+  },
+  "integrationsTableEmptyStateBody": {
+    "defaultMessage": "Connect to webhooks and external applications.",
+    "description": "Integrations Empty State body"
+  },
+  "notifications": {
+    "defaultMessage": "Notifications",
+    "description": "Application name"
+  }
+}
```

Comment:
Is this build file required in the commit?

Reply:
yes

Reply:
Ok. I think we could check how to integrate the generation of this file in the build process (in a separate PR). 
I suppose that is manually updated, right? At least having a check to validate that is "updated" (again, in a separate PR).

Reply:
We can change the location if yah want, but its the where the extracted strings are stored, from which the translations.json is built

Reply:
My only concern is if we can end up with an old build Messages.json file. Not updated because dev forgot to run the `translations` command. (IIUC)
It can be reviewed on a followup.

Reply:
yeah that will take a lil diligence, running translate whenever yah add new strings, we could hijack start to also do it... but the string adding isn't always a thing done, ALSO until we're seriously seriously using intl, kinda doesn't quite matter if they get outta date 🤷